api/users/services.py

```diff
@@ -16,6 +17,7 @@ def create_user(
     username: str,
     password: str,
     email: Optional[str] = None,
+    fcm_token: Optional[str],
```

Comment:
any Optional field should have default value

Reply:
Yeah, I was made it optional to test but then forgot to change it 😬